react-native-sdk/index.tsx

```diff
@@ -37,10 +36,19 @@ interface IAppProps {
     style?: Object;
 }
 
+/**
+ * Converts the meetingOptions domain and roomName to a URL that can be passed to the App component.
+ * @param {*} domain domain address from props.
+ * @param {*} roomName room name from props.
+ */
+const convertPropsToURL = (domain, roomName) => {
+    return `${domain}/${roomName}`;
+}
+
 /**
  * Main React Native SDK component that displays a Jitsi Meet conference and gets all required params as props
  */
-const JitsiMeet = forwardRef(({ flags, meetingOptions, style }: IAppProps, ref) => {
+const Index = forwardRef(({ flags, meetingOptions, style }: IAppProps, ref) => {
```

Comment:
Let's call this JitsiMeeting, similar to how we name it in the React SDK.

Also let's not make it the default export, make it a names one, in case we want to export more stuff later on.

Reply:
Updated